src/main/java/sirius/biz/util/ExtractedFileBuffer.java

```diff
@@ -90,11 +91,14 @@ public long getSize() {
     }
 
     /**
-     * Closes the underlying output stream.
+     * Releases all internally acquired resources.
      */
-    public void closeOutputStream() {
+    public void cleanup() {
```

Comment:
Why not implement (Auto)Closeable? Then it can be used with try-with-resources etc.

Reply:
It is an internal class. And the way the 7-ZIP binding works, we woudln't really be able to use the pattern anyway. However, using Closable is generally a good choice :)